walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,137 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void everyAttendeeIsConsideredExceptOptional() {
```

Comment:
optional: a common test method naming convention in Java is <methodName>_<expectedBehavior>. Having the method name makes it clear which method in the class is being tested. Right now, it's not an issue since you only have one (public) method, but just an FYI

https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names